swagger.yml

```diff
@@ -725,4 +735,25 @@ components:
         realmRoles:
           type: array
           items:
-            type: string
\ No newline at end of file
+            type: string
+        identifications:
+          type: array
+          items:
+            $ref: "#/components/schemas/UserIdentification"
+
+    UserIdentification:
```

Comment:
What kind of identification data this object contains? Or should it be identity?